packages/http-client/src/http-client.ts

```diff
@@ -275,7 +275,7 @@ function splitRequestConfig(requestConfig: HttpRequestConfigWithOrigin): {
 
 function logCustomHeadersWarning(customHeaders?: Record<string, string>) {
   if (customHeaders) {
-    logger.info(
+    logger.warn(
```

Comment:
Potentially this could even be debug...

Reply:
yes debug because the custom headers are sometimes used intentionally to overwrite the default. Also I think the message needs to be adjust, because it will overwrite only if the headers are set by SDK. So `custom headers with highest priorites are: XYZ, they will overwrite headers set by the SDK` or something.